dev/integration_tests/semantics/test_driver/android_testing.dart

```diff
@@ -0,0 +1,512 @@
+  static const AndroidSemanticsAction previousHtmlElement =
+      const AndroidSemanticsAction._(0x800);
+
+  /// Taken from [AccessibilityAction.ACTION_SCROLL_FORWARD].
+  static const AndroidSemanticsAction scrollForward =
+      const AndroidSemanticsAction._(0x1000);
+
+  /// Taken from [AccessibilityAction.ACTION_SCROLL_BACKWARD].
+  static const AndroidSemanticsAction scrollBackward =
+      const AndroidSemanticsAction._(0x2000);
+
+  /// Taken from [AccessibilityAction.ACTION_CUT].
+  static const AndroidSemanticsAction cut =
+      const AndroidSemanticsAction._(0x4000);
+
+  /// Taken from [AccessibilityAction.ACTION_COPY].
+  static const AndroidSemanticsAction copy =
+      const AndroidSemanticsAction._(0x8000);
+
+  /// Taken from [AccessibilityAction.ACTION_PASTE].
+  static const AndroidSemanticsAction paste =
+      const AndroidSemanticsAction._(0x10000);
+
+  /// Taken from [AccessibilityAction.ACTION_SET_SELECTION].
+  static const AndroidSemanticsAction setSelection =
+      const AndroidSemanticsAction._(0x20000);
+
+  /// Taken from [AccessibilityAction.ACTION_EXPANwD].
+  static const AndroidSemanticsAction expand =
+      const AndroidSemanticsAction._(0x40000);
+
+  /// Taken from [AccessibilityAction.ACTION_COLLAPSE].
+  static const AndroidSemanticsAction collapse =
+      const AndroidSemanticsAction._(0x80000);
+
+  @override
+  String toString() {
+    switch (id) {
+      case 0x1:
+        return 'AndroidSemanticsAction.focus';
+      case 0x2:
+        return 'AndroidSemanticsAction.clearFoucs';
+      case 0x4:
+        return 'AndroidSemanticsAction.select';
+      case 0x8:
+        return 'AndroidSemanticsAction.clearSelection';
+      case 0x10:
+        return 'AndroidSemanticsAction.click';
+      case 0x20:
+        return 'AndroidSemanticsAction.longClick';
+      case 0x40:
+        return 'AndroidSemanticsAction.accessibilityFocus';
+      case 0x80:
+        return 'AndroidSemanticsAction.clearAccessibilityFocus';
+      case 0x100:
+        return 'AndroidSemanticsAction.nextAtMovementGranularity';
+      case 0x200:
+        return 'AndroidSemanticsAction.previousAtMovementGranularity';
+      case 0x400:
+        return 'AndroidSemanticsAction.nextHtmlElement';
+      case 0x800:
+        return 'AndroidSemanticsAction.previousHtmlElement';
+      case 0x1000:
+        return 'AndroidSemanticsAction.scrollForward';
+      case 0x2000:
+        return 'AndroidSemanticsAction.scrollBackward';
+      case 0x4000:
+        return 'AndroidSemanticsAction.cut';
+      case 0x8000:
+        return 'AndroidSemanticsAction.copy';
+      case 0x10000:
+        return 'AndroidSemanticsAction.paste';
+      case 0x20000:
+        return 'AndroidSemanticsAction.setSelection';
+      case 0x40000:
+        return 'AndroidSemanticsAction.expand';
+      case 0x80000:
+        return 'AndroidSemanticsAction.collapse';
+      default:
+        return 'AndroidSemanticsAction.unknown';
```

Comment:
Does unknown actually exist? Or is this more of an error case, meaning if you get this we forgot to include an action in the above list? If it is the latter, should this maybe throw to indicate the error condition?